website/project/views/node.py

```diff
@@ -203,16 +203,25 @@ def project_new_node(auth, node, **kwargs):
                 http.BAD_REQUEST,
                 data=dict(message_long=e.message)
             )
-
+        redirect_url = new_component.url
         message = (
-            'Your component was created successfully. You can keep working on the component page below, '
+            'Your component was created successfully. You can edit contributor permissions below, '
```

Comment:
This is the message we deliver when they're *not* adding contributors. Why are we changing it here?

![screen shot 2016-02-11 at 3 08 07 pm](https://cloud.githubusercontent.com/assets/134455/12989053/8d4c9354-d0d1-11e5-8a62-aaede05ff69e.png)